src/components/view/appView.ts

```diff
@@ -20,6 +23,12 @@ export class AppView {
         const values = data?.sources ? data?.sources : [];
```

Comment:
Not obvious data type of values variable. Please use explicit type

Reply:
Added types to all variables (should i do it when type written just before variable definition? Like `const newsItemTemp: HTMLTemplateElement = <HTMLTemplateElement>document.querySelector('#newsItemTemp');`

Reply:
It is optional in a real project but now we have a task that needs all types to be written